plugins/defaultplugins/plugin_impl_vpp.go

```diff
@@ -231,6 +265,13 @@ func (plugin *Plugin) Init() error {
 
 	return nil
 }
+func (plugin *Plugin) resolveResyncStrategy(strategy string) string {
+	if strategy == defaultResync || strategy == interfaceBased || strategy == skipResync {
```

Comment:
please rename "interfaceBased" to "optimizeColdStart"
please rename "defaultResyn" to "fullResync"

the "skipResync" should not be a strategy but instead of that it should be second configuration parameter/flag "skipResyncAtStartup"